lib/alephant/lookup/lookup_helper.rb

```diff
@@ -21,15 +23,17 @@ def initialize(lookup_table)
       end
 
       def read(id, opts, batch_version)
-        LookupQuery.new(lookup_table.table_name, id, opts, batch_version).run!.tap do
-          logger.info(
-            "event"        => "LookupQuery",
-            "tableName"    => lookup_table.table_name,
-            "id"           => id,
-            "opts"         => opts,
-            "batchVersion" => batch_version,
-            "method"       => "#{self.class}#read"
-          )
+        LookupCache.new(@config).get(component_cache_key(id, opts, batch_version)) do
```

Comment:
I placed the cache in the lookup helper. I wasn't sure of the best place to add it, but thought the query was too low down. I am hoping to store and retrieve a LookupLocation object from the cache, so it is consistent with the object LookupQuery returns, and also the same return type as the write method.